docs/fastdds/env_vars/env_vars.rst

```diff
@@ -13,45 +15,51 @@ This is the list of environment variables that affect the behavior of *Fast DDS*
     |                                                                  |
     |    export FASTRTPS_DEFAULT_PROFILES_FILE=/home/user/profiles.xml |
     +------------------------------------------------------------------+
     | **Windows**                                                      |
     +------------------------------------------------------------------+
-    | .. code-block::                                                  |
+    | .. code-block:: bash                                             |
     |                                                                  |
     |    set FASTRTPS_DEFAULT_PROFILES_FILE=C:\profiles.xml            |
     +------------------------------------------------------------------+
 
 
-``ROS2_AUTO_CLIENT_SERVER``
-    When this variable is set, it automatically activates the :ref:`Server-Client Discovery<discovery_server>`.
-    The value of the variable must represent the locator of the server,
-    in the form of the IP address (e.g., '192.168.2.23') or address-port pair (e.g., '192.168.2.23:24353').
-    If no port is specified, the default port 11311 is used.
-
-    If the specified address is a local address of the current machine, it will start the
-    :ref:`dds_layer_domainParticipant` as a server.
-    If the address is a remote one, or if there is already a server at the given port,
-    it will start the DomainParticipant as a client,
-    connecting to the server at the specified locator.
-
-    +--------------------------------------------------------------+
-    | **Linux**                                                    |
-    +--------------------------------------------------------------+
-    | .. code-block::                                              |
-    |                                                              |
-    |    export ROS2_AUTO_CLIENT_SERVER=192.168.2.23:24353         |
-    +--------------------------------------------------------------+
-    | **Windows**                                                  |
-    +--------------------------------------------------------------+
-    | .. code-block::                                              |
-    |                                                              |
-    |    set ROS2_AUTO_CLIENT_SERVER=192.168.2.23:24353            |
-    +--------------------------------------------------------------+
-
-
-``FASTDDS_AUTO_CLIENT_SERVER``
-    An alias for ``ROS2_AUTO_CLIENT_SERVER``.
-    If both variables are defined, only the value of ``ROS2_AUTO_CLIENT_SERVER`` is used.
+``ROS_DISCOVERY_SERVER``
+    Setting this variable configures the participant as a client of
+    :ref:`Server-Client Discovery<discovery_server>` provided its
+    :ref:`dds_layer_domainParticipant`'s |discoveryProtocol| setting has been left configured as default.
+    More information on configuring the DomainParticipant can be found in :ref:`participantprofiles`.
```

Comment:
```suggestion
```

Reply:
Write a warning note here saying that the environment variable is only used in case the `discoveryProtocol` is set to SIMPLE. In any other case, the environment variable has no effect.